addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -29,6 +29,7 @@ class ChannelMember(models.Model):
     custom_channel_name = fields.Char('Custom channel name')
     fetched_message_id = fields.Many2one('mail.message', string='Last Fetched', index="btree_not_null")
     seen_message_id = fields.Many2one('mail.message', string='Last Seen', index="btree_not_null")
+    read_message_id = fields.Many2one('mail.message', string='Last Read', default=lambda s: s.seen_message_id, index="btree_not_null")
```

Comment:
I agree for the need of a new field, but I find the current naming confusing.
Seen and read are for me the same idea.

What should distinguish them is what they actually represent ("separator"/"seen for self"/"manual seen" vs "seen for others"/"true seen").

Reply:
It's the default parameter enough on the field def to set the value initially? Before any user action, the values are supposed to be the same.

Reply:
Default is only when creating new records, and only from the ORM.

Reply:
Ok then i'll create an update PR. We just have to agree on the field's name :)

Reply:
For now I don't have a better suggestion than `new_message_separator`.